products/early_access_features/frontend/earlyAccessFeatureLogic.ts

```diff
@@ -81,6 +84,44 @@ export const earlyAccessFeatureLogic = kea<earlyAccessFeatureLogicType>([
                 return result
             },
         },
+
+        personsCount: [
+            null as [number, number] | null,
+            {
+                loadEarlyAccessFeatureSuccess: async (_, breakpoint) => {
+                    // Should exist because it was a success, but Typescript doesn't know that
+                    if (!values.earlyAccessFeature || !('feature_flag' in values.earlyAccessFeature)) {
+                        return null
+                    }
+
+                    if (values.personsCount === null) {
+                        await breakpoint(100)
+                    } else {
+                        await breakpoint(1000)
+                    }
```

Comment:
Maybe a comment to explain why we wait here?

Reply:
Sure! It's to avoid rerender problems :). Will add a comment

Reply:
Actually, I'll just remove this, it works well without it locally